src/app.js

```diff
@@ -463,6 +465,16 @@
 				// find if this is our current page
 				// current route found ?
 				if (!!~App.pages._matchRoute(currentRouteUrl, this.routes())) {
+					if (!!currentPage) {
+						App.log({
+							args: ['Previous current page will be changed', {
+								currentPage: currentPage,
+								previousPage: previousPage,
+								newPage: this
```

Comment:
Should we use newCurrentPage param instead of newPage?

Reply:
Ok